Automattic/wp-calypso
# Email stats: make default date the post date

#### Proposed Changes

Currently the email open stats take the current date as the default date when opening them. This doesn't make much sense, because most opens happen on the day and the days after an email is sent. Therefore, this PR shows the post date as the first date in the chart.

#### Testing Instructions

* Apply this PR
* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats`
* When viewing open email stats, the first date should be the post date:  
 
<img width="1123" alt="CleanShot 2023-01-13 at 14 14 12@2x" src="https://user-images.githubusercontent.com/528287/212328308-8bf64089-e0d7-4a33-916d-69cf7ee65191.png">

* Also check that Jetpack Stats charts still behave as before.

#### Known limitation
If you view stats from more than `MAX_BARS` days (depending on the width of your screen this is 26 or less) ago, and use the arrows to go forward MAX_BARS + 1 days, it will still show the initial date. Will fix this in a future PR.

#### Pre-merge Checklist
